### ISSUE TITLE
Never reseting the game

### ISSUE THREAD

**Issue by el3ment** [kihyunwon/dqn_atari#1]
I'm not sure if I'm missing something -- but you don't seem to be restarting the game if terminal=True during the "initializing replay" and actual training

**Comment by kihyunwon** [comment]
@el3ment yes, you are right. It should restart the game when reaching terminal state during training.

**Comment by el3ment** [comment]
Did it automatically reset for Space Invaders? I'm surprised you were
getting any learning.
